### ISSUE TITLE
Recurring payment plan not working

### ISSUE THREAD

**Issue by AshokRadadiya** [PaystackHQ/paystack-ios#46]
iOS PaystackSDK-3.0.15 not working when set Recurring plan payment method.

**Comment by dilipiOSDeveloper** [comment]
Have you integrated PayStack in iOS @AshokRadadiya

**Comment by AshokRadadiya** [comment]
> Have you integrated PayStack in iOS @AshokRadadiya

Yes, We have integrated PayStack in iOS. Recurring working fine but some devices not work properly and crashed App.
